### PR TITLE
Clean 3+ years old auto-updates

### PR DESCRIPTION
**Pull request by Alkarex**
* Minor auto-update in version 1.10.0 (3 years old) https://github.com/FreshRSS/FreshRSS/pull/1750
    * Causing slight problems, related to https://github.com/FreshRSS/FreshRSS/issues/3552
* Auto-update for temporary table in version 1.7.0 (4 years old) https://github.com/FreshRSS/FreshRSS/pull/1470
    * Just because it was older than the one just above

### CODE REVIEW

File: app/Models/CategoryDAO.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function updateCategory($id, $valuesTmp) {
 		} else {
 			$info = $stm == null ? $this->pdo->errorInfo() : $stm->errorInfo();
 			if ($this->autoUpdateDb($info)) {
-				return $this->updateCategory($valuesTmp);
+				return $this->updateCategory($id, $valuesTmp);
```
Comment (Alkarex) [inline app/Models/CategoryDAO.php:151]: And a small auto-update bug fix at the same time

**Review by Frenzie — COMMENTED** [review]
I forget, can you go 1.7 → 1.8 → 1.9 or does it always try to go straight to the latest

**Comment by Alkarex** [comment]
To the latest @Frenzie . And actually, I am not sure I like this PR after all, especially for a .1 release. Will do something else instead

**Comment by Alkarex** [comment]
Revived in https://github.com/FreshRSS/FreshRSS/pull/5649

**Pull request closed without merging**